src/data_cleaning.R

```diff
@@ -0,0 +1,54 @@
+#preamble
+rm(list=ls()) # clear vars
+cat("\014") # clear console
+library(tidyverse)
+library(lubridate)
+library(ggplot2)
+library(tsibble)
+
+
+setwd("C:\\Users\\Yamni\\Desktop\\TNMD-mini-hackathon\\air_quality_data")
+
+
+#read data
+data<-read.csv("Air_Quality_Monitoring_Data.csv")
```

Comment:
Recommend using `readr::read_csv` when using `dplyr` functions to shape data, as it automatically reads data in as a tibble.